easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20230823-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Bundle'
+
+name = 'Jupyter-bundle'
+version = '20230823'
+
+homepage = "https://jupyter.org/"
+
+description = """
+ This bundle collects a range of Jupyter interfaces (Lab, Notebook and nbclassic),
+  extensions (Jupyter Server Proxy, Jupyter Resource Monitor, Jupyter Lmod) and
+  the JupyterHub.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+dependencies = [
+    ('JupyterHub', '4.0.2'),
+    ('JupyterLab', '4.0.5'),
+    ('JupyterNotebook', '7.0.2'),
+    ('nbclassic', '1.0.0'),
+    ('jupyter-server-proxy', '4.0.0'),
+    ('jupyterlmod', '4.0.3'),
+    ('jupyter-resource-usage', '1.0.0'),
+]
```

Comment:
@casparvl Please add a `moduleclass` here (why the hell doesn't the easyconfigs test suite trip on that, huh...)

Reply:
Done